src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import duke.customexception.IllegalInputException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+    public static void printLineSeparator() {
+        System.out.println("---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------");
+    }
+
+    public static void printGreetings() {
+        String LOGO = " _           _        \n"
+                + "| |    _   _| | _____ \n"
+                + "| |   | | | | |/ / _ \\\n"
+                + "| |_  | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println("Hello! I'm Luke");
+        System.out.println("What can I do for you?");
+        printLineSeparator();
+    }
+
+    public static boolean isTask(String line) throws IllegalInputException {
+        if (!(line.startsWith("todo") || line.startsWith("event") || line.startsWith("deadline"))) {
+            throw new IllegalInputException();
+        }
+        return true;
+    }
+
+    public static void main(String[] args) throws IOException {
+        printGreetings();
+        TaskManager list = new TaskManager();
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        File taskFile = createFile();
+        readFile(list, taskFile);
+        while (!line.equals("bye")) {
+            printLineSeparator();
+            try {
+                if (line.equals("list")) {
+                    list.printList();
+                } else if (line.startsWith("done")) {
+                    list.checkOffTask(line);
+                } else if (line.startsWith("delete")) {
```

Comment:
similar here, perhaps move all fixed strings like bye, list, done and delete to constants.